src/cli/error.rs

```diff
@@ -259,6 +262,9 @@ impl From<CBShellError> for ShellError {
             CBShellError::LLMApiKeyMissing {provider} => {
                 spanned_shell_error(format!("api_key required to use {} models", provider), Some("Define an api_key in the config/credentials file".to_string()), None)
             }
+            CBShellError::CreatedClusterNotRegistered {inner_error} => {
+                spanned_shell_error(format!("Cluster created but not registered: {}", inner_error), None, None)
```

Comment:
I think this should be updated to add a bit more information about what "not registered" means.